Dockerfile-base

```diff
@@ -1,7 +1,7 @@
-FROM quay.io/deis/base:0.3.0
+FROM quay.io/deis/base:v0.3.6
```

Comment:
Good idea. I'll move the deis v0.3.6 security update to a [mozmar/base](https://github.com/mozmar/docker-base) PR.